### ISSUE TITLE
Add option to remove other loggers 

### ISSUE THREAD

**Issue by 304NotModified** [NLog/NLog.Web#192]
On by Default

```
.ConfigureLogging((context, logging) =>
{
    // clear all previously registered providers
    logging.ClearProviders();

    // now register everything you *really* want
    // …
});
```

**Comment by snakefoot** [comment]
Alternative have option that registers the NLog LoggerFactory. Then all other providers will automatically be removed.

It will also have the extra side-effect that only NLog-Logging-Rules has effect, so it is simpler to diagnose (Not being confused by environment-specific appsettings.json files).

**Comment by snakefoot** [comment]
Resolved by  #657 that adds the option `ReplaceLoggerFactory` (And `RemoveLoggerFactoryFilter` that will become enabled by default with ver. 5)
